site/docs/pages/App.tsx

```diff
@@ -32,7 +32,9 @@ export default function App({ children }: { children: ReactNode }) {
   return (
     <WagmiProvider config={wagmiConfig}>
       <QueryClientProvider client={queryClient}>
-        <div style={{ display: 'flex', flexDirection: 'column' }}>{children}</div>
+        <div style={{ display: 'flex', flexDirection: 'column' }} className="ock-docsite-content">
```

Comment:
Do we want all our demo to shocase Inter?

Reply:
Great question. Let's check in with design tomorrow